### ISSUE TITLE
 Unknown SSL protocol error on build_on_success

### ISSUE THREAD

**Issue by cschlegelmilch** [Shippable/support#4789]
https://app.shippable.com/github/bucketduck/doobi/runs/364/1/console

We've had a curl call in the build_on_success step start failing with a `Unknown SSL protocol error in connection to api.ghostinspector.com:443 ` error for no apparent reason.  This has worked, unchanged, for over a year.  The curl call seems to resolve manually when tried locally.  Has anything changed in the build environment that may have caused this?

**Comment by a-murphy** [comment]
The image you are using to run builds in your project hasn't changed.  Since the last success I was able to find for that command was in August, I suspect that the site you are trying to reach has changed.  For instance, the certificates could have been updated to one that the older image can't verify.  Since you are using a very old image, I would suggest [updating your subscription node pool to a newer machine image](http://docs.shippable.com/platform/runtime/machine-image/ami-overview/#changing-the-node-pool-machine-image) to use newer build images.   Updating your `shippable.yml` to the [newer format](http://docs.shippable.com/ci/yml-structure/) or setting one of the [newer images](http://docs.shippable.com/platform/runtime/machine-image/ami-v724/) in the `build_image` YML line will make sure it uses the new images.
